.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: rust
 sudo: false
 rust:
-  - 1.0.0
+  - stable
```

Comment:
Can you leave `1.0.0` here? The intention with that is to ensure that this continues to always compile on 1.0.0 Rust